bonnie-businessrules/src/main/java/com/cgi/hexagon/businessrules/order/Order.java

```diff
@@ -6,14 +6,14 @@
 
 public class Order {
 
-    long id;
-    String shopId;
-    String goodsId;
-    int quantity;
-    Status status;
-    String assembler;
+    private long id;
+    private String shopId;
+    private String goodsId;
+    private int quantity;
+    private Status status;
+    private String assembler;
 
-    String trackingNr;
+    private OrderMetadata metadata;
```

Comment:
TrackingNr  is similar to shipping details, Bonnie has to store without modification like shipping details. 
Can we separate from timestamp or state?

Reply:
What do you mean by separating them? I think they would fit into metadata. Although I can make a dedicated property, just please describe it.